plugin.video.yelo/.github/workflows/addon-check.yml

```diff
@@ -0,0 +1,39 @@
+# This workflow performs the required add-on checks for a submission
```

Comment:
Please drop from the submission. If using ci/cd add it to gitattributes as export-ignore